app/src/main/res/layout/fragment_main.xml

```diff
@@ -21,33 +19,69 @@
 
     </androidx.coordinatorlayout.widget.CoordinatorLayout>
 
-    <FrameLayout
-        android:id="@+id/fragment_main_nav_tab_container"
+    <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="?attr/actionBarSize"
-        android:layout_gravity="bottom"
-        android:background="?attr/nav_tab_background_color"
-        android:elevation="6dp">
+        android:layout_height="wrap_content"
+        android:baselineAligned="false"
+        android:orientation="horizontal">
```

Comment:
The button bar no longer has a shadow below it. This view should probably have an `elevation` of 6dp, instead of its child views.

Reply:
Awesome!

Reply:
the NavTabLayout still needs it, but other than that it worked!